AAI/AAIConnectProfile.md

```diff
@@ -803,6 +814,209 @@ following:
     encrypted at rest and follow best practices to limit the ability of
     administrators from decrypting this content.
 
+### Appendix - Sequence Diagrams
+
+The following sequence diagrams are included to help explain the intended flows
+documented in this specification. However the diagrams are non-normative - if there are
+any discrepancies between the diagrams and the text of the specification, the text
+of the specification will take precedence.
+
+#### Callback (Userinfo) Flow
+
+The flow as used by Elixir uses the initial *Passport-Scoped OAuth Access Token* as
+a token handed to downstream resource servers. These servers can use this token, in conjunction
+with a callback to the `/userinfo` endpoint of the broker, to obtain the *Passport* content in
+JSON format.
+
+{% plantuml %}
+
+hide footbox
+skinparam BoxPadding 10
+skinparam ParticipantPadding 20
+
+box "Researcher"  #eee
+actor       "User Agent"                as user
+participant Client                      as client
+end box
+
+box "AAI"
+participant Broker                      as broker
+collections "IdP"                       as idps
+end box
+
+box "Data Owner"
+collections "Visa Issuer(s)"            as issuers
+end box
+
+box "Data Holder"
+participant "Clearing House"            as clearing
+participant "Data"                      as data
+end box
+
+==OIDC==
+
+ref over user, client, broker, idps
+OIDC flow results in the client holding a *Passport-Scoped OAuth Access Token*.
+end ref
+
+==Use==
+
+user -> client : User instructs client\nregarding data access
+client -> clearing : Client asks for data
+note over client, clearing  #CCCCCC
+{
+   Authorization: Bearer <Passport-Scoped OAuth Access Token>
+}
+end note
+
+clearing -> broker : Please give me details of the passport
+note over broker, clearing  #CCCCCC
+{
+  Authorization: Bearer <Passport-Scoped OAuth Access Token>
+}
+end note
+
+broker -> issuers : (optional) Fetch signed visa(s) for user
+note over broker, issuers
```

Comment:
This note showing returned visas should be **after** the **Return signed visa(s) for user** synchronous message because it describes its content.